### PR TITLE
Three sum Problem

### PR DESCRIPTION
**Pull request by devRawnie**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Implemented the three sum problem in Python using the two-pointer technique

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#7045

### CODE REVIEW

**Comment by devRawnie** [comment]
@ankitaggarwal23 @HarshCasper @bhav09 @gat786 @anushkrishnav
I have created a pull request with the complete code. Kindly review the same.

File: Python/search/three_sum_problem.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    Sample Input 2: 
+    12 3 4 1 6 9    # List of numbers
+    26              # Sum
+
+    Sample Output 1:
+    No triplet found with sum: 24
+
+    Time Complexity:
+    For sorting:                O(N.log N)
+    For two-pointer technique:  O(N^2)
```
Comment (ankitaggarwal23) [inline Python/search/three_sum_problem.py]: Remove extra test case one is enough

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Write the outputs in the way your code is producing.

File: Python/search/three_sum_problem.py (hunk 1 above)
Comment (devRawnie) [reply Python/search/three_sum_problem.py]: I have removed the extra test case. Also changed the test case to match the format my code is following.

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by ankitaggarwal23** [comment]
@all-contributors bot please add @devRawnie for code

**Pull request merged**